File: PolyPaint_ClientLourd/PolyPaint/Models/Stats/UserGameStats.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PolyPaint.Models.Stats
{
    public class UserGameStats
    {
        [JsonProperty(PropertyName = "rounds_played")]
        public int RoundsPlayed { get; set; }

        [JsonProperty(PropertyName = "victories")]
        public int Victories { get; set; }

        [JsonProperty(PropertyName = "failures")]
        public int Failures { get; set; }

        [JsonProperty(PropertyName = "rounds_avg_time")]
        public double RoundAvgTime { get; set; }

        [JsonProperty(PropertyName = "total_time_played")]
        public double TotalTimePlayed { get; set; }

        [JsonProperty(PropertyName = "total_games_played")]
        public double TotalGamePlayed { get; set; }

        public UserGameStats()
        {
            RoundsPlayed = 0;
            Victories = 0;
            Failures = 0;
            RoundAvgTime = 0;
            TotalTimePlayed = 0;
            TotalGamePlayed = 0;
        }

    }
}

File: PolyPaint_Server/src/Models/UserGameStats.ts
import { Base, IBase } from "./Base";

export interface IUserGameStats extends IBase {
    rounds_played: number
    victories: number
    failures: number
    rounds_avg_time: number
    total_time_played: number,
    total_games_played: number
}

export class UserGameStats extends Base implements IUserGameStats, IBase {
    constructor(
        public rounds_played: number = 0,
        public victories: number = 0,
        public failures: number = 0,
        public rounds_avg_time: number = 0,
        public total_time_played: number = 0,
        public total_games_played: number = 0
    ) {
        super()
    }
}